### PR TITLE
Awesome books: with ES6

### PR DESCRIPTION
**Pull request by maths29**
in this project I:
- Create a separate file for each module in the modules directory.
- Import modules into the index.js file.
- Refactor your functions to arrow functions.
- Display the current date below the navigation bar
- using Luxon

Marco Lizza
  [12:12 PM](https://microverse-students.slack.com/archives/C47D4N35M/p1686654751539179)
Hey Micronauts @channel! :wave:
I hope you're doing well! We are aware that most of you are experiencing Webhint issues, we are currently working on a solution:male-technologist:.
For now, please go ahead and request for review if you only have the Webhint error on your PR. Our CRs would be happy to review your PRs :slightly_smiling_face:.
Feel free to share this announcement on your PR for the CR to see:bangbang:
Thank you for your understanding.

### CODE REVIEW

File: modules/ENG.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  contact.classList.remove('active');
+  homeScreen.classList.add('remove-content');
+}
+export function displayContactSection() {
+  contact.classList.add('active');
+  bookslisth2.classList.remove('active');
+  BooksContainer.classList.remove('active');
+  formInfo.classList.remove('active');
+  homeScreen.classList.add('remove-content');
+}
```
Comment (AmaduKamara) [inline modules/ENG.js]: - [ ] Kindly use arrow function syntax for all your functions as required. It's standard in ES6 to use arrow function syntax for your functions. Please modify your functions to arrow function syntax 👍🏽

File: modules/addAndRemoveBook.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    `;
+
+  BooksContainer.appendChild(booksInfo);
+};
+
+export function removeBookFromPage(target) {
+  if (target.classList.contains('remove-this-book')) {
+    target.parentElement.remove();
+  }
+}
```
Comment (AmaduKamara) [inline modules/addAndRemoveBook.js]: - [ ] Kindly use arrow function syntax for all your functions as required. It's standard in ES6 to use arrow function syntax for your functions. Please modify your functions to arrow function syntax 👍🏽

File: modules/localStorage.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  const newID = Number(idd);
+  for (let i = 0; i < Books.length; i += 1) {
+    if (Books[i].id === newID) {
+      k = i;
+      break;
+    }
+  }
+  Books.splice(k, 1);
+  localStorage.setItem('BooksInfo', JSON.stringify(Books));
+}
```
Comment (AmaduKamara) [inline modules/localStorage.js]: - [ ] Kindly use arrow function syntax for all your functions as required. It's standard in ES6 to use arrow function syntax for your functions. Please modify your functions to arrow function syntax 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️
 
Hi @maths29,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights:

✅ You used Gitflow correctly 👍🏽
✅ The Linter checks passed successfully 👍🏽
✅ The design looks nice 👍🏽

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by wayungi — APPROVED** [review]
Hi @maths29 , 

image

Great work, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Remember: Every finish line 🏁  is the beginning of a new race 🐎** 

## Highlights
- [ ] All requested changes implemented ✔️ 
- [ ]  Correct GitHub flow ✔️ 
- [ ] Descriptive PR body ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**